java/src/test/java/tokens/TestTokens.java

```diff
@@ -166,10 +166,12 @@ private static Stream<Arguments> tokenizationTypeAndArchived() {
                 arguments(TokenType.RANDOMIZED, false),
                 arguments(TokenType.PCI, true),
                 arguments(TokenType.PCI, false),
-                arguments(TokenType.PCI_ONEWAY, true),
-                arguments(TokenType.PCI_ONEWAY, false),
+                // arguments(TokenType.PCI_ONEWAY, true),
+                // arguments(TokenType.PCI_ONEWAY, false),
                 arguments(TokenType.DETERMINISTIC, true),
                 arguments(TokenType.DETERMINISTIC, false)
+
+                // Most of the tests are using detokenize / rotate tokens but this can't work on TokenType.PCI_ONEWAY
```

Comment:
please create a new Stream of arguments reversibleTokenTypes to contain only the relevant token types